app/exporters/odt/cv.rb

```diff
@@ -78,29 +86,52 @@ def insert_personalien(report)
     # rubocop:enable Metrics/AbcSize
 
     def insert_competences(report)
-      competences_list = if include_core_competences_and_skills?
-                           [core_competences_list, competence_notes_list].flatten
-                         else
-                           [competence_notes_list].flatten
-                         end
-      report.add_table('COMPETENCES', competences_list, header: true) do |t|
+      interest_skills(report) if include_skills_by_interests?
+      core_competences(report)
+    end
+
+    def interest_skills(report)
+      interest_value = skill_interest_value
+      interest_skills_list = skills_by_interest_value(interest_value)
+      report.add_table('COMPETENCES', interest_skills_list, header: true) do |t|
         t.add_column(:category, :category)
         t.add_column(:competence, :competence)
       end
     end
 
-    def core_competences_list
-      core_competence_skill_ids = person.people_skills.where(core_competence: true).pluck(:skill_id)
+    def skills_by_interest_value(interest_value)
+      interest_skills_ids = skills_by_interest(interest_value).pluck(:skill_id)
+      competences_list(interest_skills_ids)
+    end
+
+    def competences_list(competences_ids)
       Category.all_parents.map do |parent_c|
         skills = Skill.joins(:category)
-                      .where(categories: { parent_id: parent_c.id }, id: core_competence_skill_ids)
+                      .where(categories: { parent_id: parent_c.id }, id: competences_ids)
                       .pluck(:title)
         next if skills.blank?
 
         { category: parent_c.title, competence: skills.join(', ') }
       end.compact
     end
 
+    def core_competences(report)
```

Comment:
This should probably be called `insert_core_competences` as you actually insert them into the odt here